### PR TITLE
Add codesign for Windows

### PR DESCRIPTION
**Pull request by aguycalled**

### CODE REVIEW

**Comment by chasingkirkjufell** [comment]
Great to see. Will test the gitian builds when they are ready.

**Comment by mxaddict** [comment]
I'm not familiar with how the signing works on windows either, but will test the gitian build once ready.

**Comment by aguycalled** [comment]
Windows signed binary: https://build.nav.community/binaries/win-codesign/navcoin-6.0.0-win64-setup.exe

File: contrib/gitian-descriptors/gitian-win-signer.yml [hunk 1]
```diff
@@ -1,38 +1,46 @@
...
-  echo "f9a8cdb38b9c309326764ebc937cba1523a3a751a7ab05df3ecc99d18ae466c9  osslsigncode-1.7.1.tar.gz" | sha256sum -c
-  echo "a8c4e9cafba922f89de0df1f2152e7be286aba73f78505169bc351a7938dd911  osslsigncode-Backports-to-1.7.1.patch" | sha256sum -c
+  mkdir -p signature
+  tar -xzf signature-win.tar.gz -C signature
+
+  echo "5a60e0a4b3e0b4d655317b2f12a810211c50242138322b16e7e01c6fbb89d92f  osslsigncode-2.0.tar.gz" | sha256sum -c
 
   mkdir -p ${UNSIGNED_DIR}
-  tar -C ${UNSIGNED_DIR} -xf navcoin-win-unsigned.tar.gz
+  tar -C ${UNSIGNED_DIR} -xf bitcoin-win-unsigned.tar.gz
```
Comment (mxaddict) [inline contrib/gitian-descriptors/gitian-win-signer.yml:34]: I think this line needs to be changed back to `navcoin`

**Review by mxaddict — APPROVED** [review]

**Comment by chasingkirkjufell** [comment]
Still getting warnings when trying to install it on Windows 10.

**Comment by mxaddict** [comment]
The warning I was getting was because the signature used was not recognized
on windows.

But the code to sign the binary is working as the signed binaries alex sent
did have the correct sigs attached to it.

**Comment by navbuilder** [comment]
A new build of 8fa2f2de170441cc87890f6073e3e04e99ecfd40 has completed succesfully!
Binaries available at https://build.nav.community/binaries/win-codesign

**Review by chasingkirkjufell — APPROVED** [review]

**Pull request merged**